packages/desktop-client/e2e/transactions.test.js

```diff
@@ -34,13 +39,13 @@ test.describe('Transactions', () => {
       debit: '12.34',
     });
 
-    expect(await accountPage.getNthTransaction(0)).toMatchObject({
-      payee: 'Home Depot',
-      notes: 'Notes field',
-      category: 'Food',
-      debit: '12.34',
-      credit: '',
-    });
+    const transaction = accountPage.getNthTransaction(0);
+    await expect(transaction.payee).toHaveText('Home Depot');
```

Comment:
Say this particular set of assertions fails on the first one...
```
await expect(transaction.payee).toHaveText('Home Depot');
```
...because the data have changed when they shouldn't have. Very likely all 5 of these assertions will eventually fail. But won't the test run wait the full assertion timeout for each following line, increasing test run time?

In these cases, would it be equally reliable to use synchronous assertions after the first one?

Reply:
No, the test fails as soon as one of the steps fails.

For example: if line `42` fails retrieving transaction `0` - it will fail on that line and NOT continue on to line `43`.